packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1065,16 +1069,25 @@ export default class NewVm extends BaseComponent {
     return name_label && template
   }
 
+  _onCpusChange = event => {
+    const value = getEventValue(event)
+    const cpusMax = Math.max(value, this.state.state.cpusMax)
+    this._setState({
+      CPUs: value,
+      cpusMax,
```

Comment:
Shouldn't `cpusMax` be set to the same value as `CPUs` whenever the user changes the `CPUs` value? Otherwise, let's say the user mistakenly types `40` for `CPUs` and then deletes the `0`, then `cpusMax` would stay at `40`.